aiter/ops/triton/mha.py

```diff
@@ -1104,6 +1140,8 @@ def forward(
         return_softmax,
         is_grad_enabled,
         config=None,
+        mapping_mode=0,
+        use_remap=True,
```

Comment:
Wouldn't we want the head-first remap as the default if it beats the previous remap method?